ts/src/pro/mexc.ts

```diff
@@ -465,7 +465,7 @@ export default class mexc extends mexcRest {
         const nonce = this.safeInteger (storedOrderBook, 'nonce');
         if (nonce === undefined) {
             const cacheLength = storedOrderBook.cache.length;
-            const snapshotDelay = this.handleOption ('watchOrderBook', 'snapshotDelay', 5);
```

Comment:
The issue was that `fetchOrderBook` was not including the count of the bidasks in the orderbook.

I thought about solving it the exchange.ts file to avoid the same error in the future on other exchanges but it would involve changing the parseOrderBook signature and change it in several places and this was the only exchange that has the issue, so for now thought it was better to solve it locally.

Also added IndexType to the transpiler